src/commands/kv/bucket/mod.rs

```diff
@@ -36,6 +36,10 @@ pub fn directory_keys_values(
         let entry = entry.unwrap();
         let path = entry.path();
         if path.is_file() {
+            if print_files {
+                message::working(&format!("Parsing {}...", path.display()));
```

Comment:
Why parsing by the way?

Reply:
because we technically don't upload until later :P by then, all the files have been batched into vecs for batched KV bulk uploads, so it's not optimal to print out the files there (we'd be iterating over the files again when we can just print out files in this initial interation).

Reply:
I see. I would be fine with `Processing x ...`. `Parsing` the file sounds odd to me, especially if we don't do it at all.

Reply:
That sounds good! Will replace `Parsing` with `Processing` :)

Reply:
We aren't parsing the files so we should change this verb.

Reply:
How about `Preparing`?